w4af/plugins/attack/db/sqlmap/LICENSE

```diff
@@ -1,7 +1,7 @@
 COPYING -- Describes the terms under which sqlmap is distributed. A copy
 of the GNU General Public License (GPL) is appended to this file.
 
-sqlmap is (C) 2006-2022 Bernardo Damele Assumpcao Guimaraes, Miroslav Stampar.
+sqlmap is (C) 2006-2023 Bernardo Damele Assumpcao Guimaraes, Miroslav Stampar.
```

Comment:
Do I assume right that you updated sqlmap? Which is the used version/git commit hash of sqlmap?

Reply:
Yup. There's a unit test we have that checks that the sqlmap version is not out of date - after 90 days we need to hit the update button. According to the merge commit (our `2a56436cea0e85eb9a4af56a1f6597aeccf3131b`) this was commit `c5d7c542a` of sqlmap (just taken from the master branch).